src/ui/dialogs/vcard.ts

```diff
@@ -30,6 +32,18 @@ export default function(contact: IContact) {
    dialog = new Dialog(content);
    dialog.open();
 
+   if (Client.getOption('showTags',false))
```

Comment:
I see no reason why groups should not always be displayed.

Reply:
OK i changed behavoir of showTags option to: true = force the text to be shown, false = circles with hover to show text